docs/source/implementations.rst

```diff
@@ -0,0 +1,211 @@
+
+* **1.x** / **2.x** / etc: The major version of a stable implementation.
+* **0.x**: The implementation is not yet stable and likely will change.
+  These pre-release projects may lack polish and may not be easy to use.
+* **WIP**: Work in progress; the implementation is not yet stable and may
+  not be usable.
+
+
+-----------------
+Smithy meta model
+-----------------
+
+.. list-table::
+    :header-rows: 1
+    :widths: 25 15 10 50
+
+    * - Project
+      - Language
+      - Status
+      - Description
+    * - :ref:`specification`
+      - N/A
+      - 1.x
+      - The Smithy specification defines Smithy and its capabilities.
+    * - `Reference implementation <https://github.com/awslabs/smithy>`_
+      - Java
+      - 1.x
+      - The reference implementation of Smithy's metamodel, transformation
+        tooling, and code generators is implemented in Java.
+    * - `Smithy Diff <https://github.com/awslabs/smithy/tree/main/smithy-diff>`_
+      - Java
+      - 1.x
+      - Smithy Diff is a tool used to compare two Smithy models to check
+        for backward compatibility issues. Smithy Diff can be run via a
+        Java library or via the Smithy CLI.
+    * - `Atelier <https://github.com/johnstonskj/rust-atelier>`_
+      - Rust
+      - WIP
+      - A Rust implementation of the Smithy IDL.
+
+
+-----------
+IDE Support
+-----------
+
+.. list-table::
+    :header-rows: 1
+    :widths: 25 15 10 50
+
+    * - Project
+      - Language
+      - Status
+      - Description
+    * - `Visual Studio Code plugin <https://github.com/awslabs/smithy-vscode>`_
+      - Java
+      - WIP
+      - A Visual Studio Code extension to recognize and highlight the
+        Smithy IDL. It can also be used as a TextMate bundle in TextMate
+        and IntelliJ using a third-party plugin.
+    * - `IntelliJ plugin <https://github.com/awslabs/smithy-intellij>`_
+      - Java
+      - WIP
+      - Smithy IntelliJ provides IDE integration for the Smithy IDL within
+        IntelliJ IDEA. It utilizes smithy-language-server for its Language
+        Server Protocol implementation.
+    * - `Smithy LSP <https://github.com/awslabs/smithy-language-server>`_
+      - Java
+      - WIP
+      - A Language Server Protocol implementation for the Smithy IDL.
+
+
+-------------
+Build tooling
+-------------
+
+.. list-table::
+    :header-rows: 1
+    :widths: 25 15 10 50
+
+    * - Project
+      - Language
+      - Status
+      - Description
+    * - `Gradle Plugin <https://github.com/awslabs/smithy-gradle-plugin>`_
+      - Java
+      - 0.x
+      - The Smithy Gradle plugin is currently the only supported way to
+        build Smithy models. `Various example projects <https://github.com/awslabs/smithy-gradle-plugin/tree/main/examples>`_
+        that show how to setup and build Smithy models can be found in the
+        project's repository.
+    * - `Smithy CLI <https://github.com/awslabs/smithy/tree/main/smithy-cli>`_
+      - Java
+      - 1.x*
```

Comment:
Extra `*`

Reply:
That was intentional to call out that it's still not really distributed or released in any way (mentioned in the description of the row).